components/merkledb/src/group.rs

```diff
@@ -91,7 +93,7 @@ where
 impl<T, K, I> Group<T, K, I>
 where
     T: Access,
-    K: BinaryKey + ?Sized,
+    K: BinaryKey + ?Sized + fmt::Debug,
```

Comment:
Is this trait bound complete necessarily?

Reply:
Forgot to remove